templates/settings.yaml.erb

```diff
@@ -30,6 +30,9 @@
 :ssl_ca_file: <%= scope.lookupvar("foreman::client_ssl_ca") %>
 :ssl_priv_key: <%= scope.lookupvar("foreman::client_ssl_key") %>
 
+# HSTS setting
+:disable_hsts: <%= scope.lookupvar("foreman::disable_hsts") %>
```

Comment:
IMHO this is a badly named variable: it should always be positive (so ```enable_hsts`` defaulting to ```true```)

Reply:
Still hasn't been merged in core - see related PR (and request change there so noone merges it)

Reply:
I thought about this and I don't think we should change it. If we default to true here and make the name e.g. `enable_hsts` that will cause any users who haven't updated their settings.yaml with the new option to default to false, thus disabling HSTS for them.